docs/server/source/production-deployment-template-tendermint/architecture.rst

```diff
@@ -0,0 +1,204 @@
+Architecture of a BigchainDB Node
+==================================
+
+A BigchainDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB, MongoDB and Tendermint
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB, Monitoring Agent and Backup Agent
+  `Kubernetes Deployments <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_.
+* MongoDB and Tendermint `Kubernetes StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_.
+* Third party services like `3scale <https://3scale.net>`_,
+  `MongoDB Cloud Manager <https://cloud.mongodb.com>`_ and the
+  `Azure Operations Management Suite
+  <https://docs.microsoft.com/en-us/azure/operations-management-suite/>`_.
+
+
+.. code:: text
```

Comment:
While this _does_ render (being just text), it's not very readable, at least not for me: the rendered result is too wide and too tall to fit on my screen so I have to scroll around.

I _do_ like the idea of having the "code" for the diagram as text, rather than creating it in some external editor and exporting an image file. I bet there's a Sphinx extension that lets you enter a diagram description/code; it then renders the result to some nice graphic.

Reply:
I see you resolved this by using `aafigure`. Cool!

:+1: